optimizely/decision_service.py

```diff
@@ -155,29 +155,55 @@ def get_variation(self, experiment, user_id, attributes, ignore_user_profile=Fal
 
     return None
 
-  def get_variation_for_layer(self, layer, user_id, attributes=None, ignore_user_profile=False):
-    """ Determine which variation the user is in for a given layer.
+  def get_variation_for_rollout(self, rollout, user_id, attributes=None, ignore_user_profile=False):
+    """ Determine which variation the user is in for a given rollout.
     Returns the variation of the first experiment the user qualifies for.
 
     Args:
-      layer: Layer for which we are getting the variation.
+      rollout: Rollout for which we are getting the variation.
       user_id: ID for user.
       attributes: Dict representing user attributes.
       ignore_user_profile: True to ignore the user profile lookup. Defaults to False.
```

Comment:
Btw, just realized, this is no longer needed because we don't call `get_variation`. You can safely remove it to avoid confusion!